loom.config

```diff
@@ -2,14 +2,14 @@
   "sdk_version": "1.1.4432",
   "executable": "Main.loom",
   "display": {
-    "width": 1024,
-    "height": 768,
+    "width": 1080,
+    "height": 1920,
```

Comment:
Should we perhaps work on a scaled down version that fits on most dev screens and then up it for finalish version?